images/macos/provision/core/dotnet.sh

```diff
@@ -19,35 +19,20 @@ chmod +x ./dotnet-install.sh
 ARGS_LIST=()
 echo "Parsing dotnet SDK (except rc and preview versions) from .json..."
 
-# TO-DO: move the list of versions to install to toolset
-if is_BigSur; then
-    DOTNET_CHANNELS=(
-    'https://raw.githubusercontent.com/dotnet/core/master/release-notes/2.1/releases.json'
-    'https://raw.githubusercontent.com/dotnet/core/master/release-notes/3.1/releases.json'
-    )
-elif is_Less_Catalina; then
-    DOTNET_CHANNELS=(
-    'https://raw.githubusercontent.com/dotnet/core/master/release-notes/2.1/releases.json'
-    )
-else
-    DOTNET_CHANNELS=(
-    'https://raw.githubusercontent.com/dotnet/core/master/release-notes/2.1/releases.json'
-    'https://raw.githubusercontent.com/dotnet/core/master/release-notes/3.0/releases.json'
-    'https://raw.githubusercontent.com/dotnet/core/master/release-notes/3.1/releases.json'
-    )
-fi
+DOTNET_VERSIONS=$(get_toolset_value '.dotnet[].versions')
```

Comment:
I think it should be `.dotnet.versions[]`

Reply:
Try to test it on https://jqplay.org/

Reply:
Thank you, fixed